var/spack/repos/builtin/packages/chapel/package.py

```diff
@@ -72,8 +72,8 @@ class Chapel(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     patch("fix_spack_cc_wrapper_in_cray_prgenv.patch", when="@2.0.0:")
     patch("fix_chpl_shared_lib_path.patch", when="@2.1:2.2 +python-bindings")
-    patch("fix_chpl_shared_lib_path_2.3.patch", when="@2.2.1: +python-bindings")
-    patch("fix_chpl_line_length.patch")
+    patch("fix_chpl_shared_lib_path_2.3.patch", when="@2.2.1:2.3 +python-bindings")
```

Comment:
To ease future maintenance, I'd like to request comments here recording where each defect was fixed, as done for the patches below. i.e. something of the form:

```suggestion
    patch("fix_chpl_shared_lib_path_2.3.patch", when="@2.2.1:2.3 +python-bindings")  # PR 26388
```

and similarly for at least the two surrounding patch lines (the prior patch line is the same PR, I have not tried to track down the PR for the subsequent patch line)